examples/gno.land/p/demo/grc/grc721/grc721_royalty_test.gno

```diff
@@ -0,0 +1,99 @@
+package grc721
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/testutils"
+	"gno.land/p/demo/users"
+	"gno.land/p/demo/ufmt"
+)
+
+func TestSetTokenRoyalty(t *testing.T) {
+	dummy := NewNFTWithRoyalty(dummyNFTName, dummyNFTSymbol)
+	if dummy == nil {
+		t.Errorf("should not be nil")
+	}
+
+	addr1 := std.Address("g1var589z07ppjsjd24ukm4uguzwdt0tw7g47cgm")
+	addr2 := std.Address("g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj")
+
+	paymentAddress := std.Address("g1var589z07ppjsjd24ukm4uguzwdt0tw7g47cgm")
+	percentage := uint64(10) // 10%
+
+	salePrice := uint64(1000)
+	expectRoyaltyAmount := uint64(100)
+
+	std.TestSetOrigCaller(std.Address(addr1)) // addr1
```

Comment:
It seems that this cast is unnecessary as addr1 is already of type `std.Address`

Reply:
i solved both at this https://github.com/gnolang/gno/pull/1962#discussion_r1593361515